src/main/java/uk/nhs/prm/repo/ehrtransferservice/services/gp2gp_messenger/Gp2gpMessengerService.java

```diff
@@ -71,7 +71,7 @@ public void sendEhrCompletePositiveAcknowledgement(UUID inboundConversationId) {
             repositoryAsid,
             record.sourceGp(),
             inboundConversationId.toString(),
```

Comment:
Doesn't this also need uppercasing?
```suggestion
            inboundConversationId.toString().toUpperCase(),
```

Reply:
Fixed